lib/checkPullRequestLabels.js

```diff
@@ -0,0 +1,40 @@
+module.exports.checkPullRequestLabels = async function(context) {
+  var pullRequest = context.payload.pull_request;
+  var pullRequestNumber = pullRequest.number;
+
+  // eslint-disable-next-line no-console
+  console.log('RUNNING LABEL CHECKS ON PULL REQUEST ' + pullRequestNumber + ' ...');
+
+  var labels = pullRequest.labels;
+  var userName = pullRequest.user.login;
+  var projectOwnerLabel;
+  var hasProjectOwnerLabel = labels.some(function(label) {
+    projectOwnerLabel = label.name;
+    return projectOwnerLabel.startsWith('PROJECT');
+  });
+
+  // If the PR has a project owner label, assign the project owner
+  // for the first-pass review of the PR and leave a top-level comment.
+  if(hasProjectOwnerLabel === true) {
```

Comment:
`if (hasProjectOwnerLabel)`

Reply:
I think we tried this when implementing the checks. It's safe to use `===` for conditionals. We use it throughout the code base.

Reply:
Yep, I agree it's safe, but when the variable is a boolean and you're comparing it to a boolean it's fine to use `if(X)` of `if(!X)` for simplicity.

Reply:
Also please add space after "if"

Reply:
Done, thanks!